test/common/trafficmanager/validator/profile.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 const (
-	timeout  = time.Second * 120 // need more time to create azure resources
+	timeout  = time.Second * 300 // need more time to create azure resources
```

Comment:
is this timeout only used for azure resources? I just reduced the timeout for most of the fleet e2e to 10 seconds.

Reply:
our experience showed that it's dangerous to relax the timeout blindly in e2e.

Reply:
The validation package is used by both it and e2e tests. need to distinguish them, and will do that in a separate PR